openfisca_uk/reforms/basic_income/simulation_4.py

```diff
@@ -50,16 +50,33 @@ class family_net_income(Variable):
     definition_period = ETERNITY
 
     def formula(family, period, parameters):
+        benefits = [
```

Comment:
Could `family_net_income` and `total_benefit_value` live somewhere outside the reform directories to be more accessible?

Reply:
Ah yeah, that's definitely a good solution - was thinking of something similar because with the current way is a bit inelegant to redefine in reforms.